### DISCUSSION TITLE
Gotek SFR1M44-DU emulator with F415 chip

### DISCUSSION THREAD

**Discussion by RaceRon** [keirf/flashfloppy#930] [Q&A]
I need the small form factor because I was trying to install it into a Tektronix TDS754D oscilloscope. This unit has the 26 pin flat flex cable
connector, the current FF download doesn't work and my question is when I looked at some of the source code( I am not a C expert) it seemed to me that some pin definitions don't match what I have found on the internet for a 26 pin interface definition. Is there a special 
FF version that is written for this specific interface?

**Comment by keirf** [comment]
See #155 where users have got this working. Some soldering is required, but the floppy pins are at least already routed correctly to the 26-pin header.

**Comment by RaceRon** [comment]
Yes this doesn’t work on my scope, where are the source files for the 26 pin interface connector? The board.c and floppy.c under gotek file in flash floppy master are for a 34 pin connector!ThanksOn Sep 3, 2024, at 2:59 AM, Keir Fraser ***@***.***> wrote:﻿
See #155 where users have got this working. Some soldering is required, but the floppy pins are at least already routed correctly to the 26-pin header.

**Reply by keirf** [reply]
There is no specific handling of the 26-pin interface in the firmware. Everything is defined in terms of the Shugart 34-pin interface.

**Comment by liveboxandy** [comment]
I have used a 26 pin interface F415 Gotek in a Tektronix TDS3052 scope. All I needed was a new reversed 26 pin cable. The only jumper required is on S1.
I followed instructions from here https://github.com/andrewpono/TDS3000-Series-Gotek-FlashFloppy

![My Scope - TDS3052 With Gotek](https://github.com/user-attachments/assets/68602562-31dd-4262-8c95-5109ce38f3b5)

**Comment by PedFastTech** [comment]
I have a TDS420A.  I have had near-complete success with the Gotek and FF.

This FF.CFG works for me; I have added display stuff because I added a 128X32 OLED to the Gotek.  

interface = ibmpc
host = pc-dos
pin34 = rdy    
pin02 = chg    
chgrst = step  
drive_select = 0
index = real
ready = real
geometry = 80,2,18,512
media_descriptor = 0xF0

display-type = auto
display-order = 1,0
display-off-secs = 255
twobutton-action = eject

debug = yes
